gensim/corpora/wikicorpus.py

```diff
@@ -526,17 +510,15 @@ def __init__(self, fname, processes=None, lemmatize=utils.has_pattern(), diction
         processes : int, optional
             Number of processes to run, defaults to `max(1, number of cpu - 1)`.
         lemmatize : bool
-            Use lemmatization instead of simple regexp tokenization?
-
-            Defaults to `True` if you have the `pattern package <https://github.com/clips/pattern>`_ installed.
+            Use lemmatization instead of simple regexp tokenization.
+            Defaults to `True` if you have the `pattern <https://github.com/clips/pattern>`_ package installed.
         dictionary : :class:`~gensim.corpora.dictionary.Dictionary`, optional
             Dictionary, if not provided,  this scans the corpus once, to determine its vocabulary
-            (this needs **really long time**).
+            (**IMPORTANT: this needs really long time**).
```

Comment:
If it's important, it should be in brackets :)

`really long time` => `a really long time`

Reply:
fixed